internal/journal/voljournal.go

```diff
@@ -261,21 +261,21 @@ func (cj *Config) Connect(monitors, namespace string, cr *util.Credentials) (*Co
 
 /*
 CheckReservation checks if given request name contains a valid reservation
-- If there is a valid reservation, then the corresponding UUID for the volume/snapshot is returned
+- If there is a valid reservation, then the corresponding ImageData for the volume/snapshot is returned
 - If there is a reservation that is stale (or not fully cleaned up), it is garbage collected using
 the UndoReservation call, as appropriate
-- If a snapshot is being checked, then its source is matched to the parentName that is provided
+- If a snapshot is being checked, then its source is matched to the snapParentName that is provided
 
 NOTE: As the function manipulates omaps, it should be called with a lock against the request name
 held, to prevent parallel operations from modifying the state of the omaps for this request name.
 
 Return values:
-	- string: Contains the UUID that was reserved for the passed in reqName, empty if
+	- ImageData: which Contains the UUID that was reserved for the passed in reqName, empty if
```

Comment:
I think the description also needs modification here, as it is also written as per a string.

Reply:
@Yuggupta27 done.. ptal.. thanks